apache2/msc_multipart.c

```diff
@@ -223,6 +260,17 @@ static int multipart_parse_content_disposition(modsec_rec *msr, char *c_d_value)
                 msr_log(msr, 9, "Multipart: Content-Disposition filename: %s",
                     log_escape_nq(msr->mp, value));
             }
+        } else if (strcmp(name, "filename*") == 0) {
```

Comment:
Why is this comparison being used twice?

Reply:
Agreed. This stuff needs to go in the first `filename*` branch.

Reply:
If we move this stuff into the first block then this block will be empty. But I'm afraid then the logic will change too much. We **MUST** check the three cases there, namely the variable `name` value is `name`, `filename` or `filename*`.

I think we should leave now as it is.

Reply:
Right. I didn't see that there are _two_ if/else structures. It looked to me like the last branch of the same if/else.